pkgs/development/python-modules/groupie/default.nix

```diff
@@ -0,0 +1,38 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchPypi,
+  pythonOlder,
+  hatchling,
+  hatch-vcs,
+}:
+
+buildPythonPackage rec {
+  pname = "groupie";
+  version = "0.2.1";
+  pyproject = true;
+
+  disabled = pythonOlder "3.13";
+
+  src = fetchPypi {
```

Comment:
switch to the github repo so that you can run tests https://github.com/mawildoer/groupie

Reply:
Done!